mpp/docs/Stubbed Projects.md

```diff
@@ -0,0 +1,36 @@
+# Stubbed Projects
+
+### List:
+- :annotation:annotation is stubbed by `:annotation:annotation-compatibility-stubs`, it depends on `androidx.annotation`
+- :collection:collection is stubbed by `:collection:collection-compatibility-stubs`, it depends on  `androidx.collection`
+
+### Purpose
+The need for "stubbed projects" appeared in this PR - https://github.com/JetBrains/compose-multiplatform-core/pull/1819
+
+YT ticket: https://youtrack.jetbrains.com/issue/CMP-7183/Infrastucture-No-need-to-publish-annotation-and-collection-libraries-anymore
+
+
+**TL;DR;**
```

Comment:
```suggestion
```
This doesn't looks like TLDR :)

If it s about the first sentence, then we should add some separator

Reply:
Right :) I started with an intention to make it TL;DR; but...

Fixed now.